src/components/TableDropdownSelect.tsx

```diff
@@ -0,0 +1,60 @@
+import { Box, styled, MenuItem, SelectChangeEvent, Select } from "@mui/material";
+import NEWMLogoSVG from "assets/images/NEWMLogo";
+import { useState } from "react";
+
+const TableDropdownSelect = () => {
+  const StyledSelect = styled(Select)(({ theme }) => ({
+    "& .css-x2bp66-MuiSvgIcon-root-MuiSelect-icon": {
+      color: theme.colors.white,
+      transform: "scale(1.2)",
+      paddingBottom: "5px",
+    },
+    "& .c.css-zsouyz-MuiSvgIcon-root-MuiSelect-icon": {
+      color: theme.colors.white,
+      transform: "scale(1.2)",
+      paddingBottom: "5px",
+    },
+    "& .MuiSelect-iconOpen": {
+      color: theme.colors.white,
+      transform: "rotate(180deg)",
+    },
+    "& .css-6t9ju4-MuiSelect-select-MuiInputBase-input-MuiOutlinedInput-input.css-6t9ju4-MuiSelect-select-MuiInputBase-input-MuiOutlinedInput-input.css-6t9ju4-MuiSelect-select-MuiInputBase-input-MuiOutlinedInput-input":
+      {},
```

Comment:
*Feedback requested comment*

was this `{}` left on by accident? also, can this selector be reduced or is the specificity required in this case?

Reply:
Actually can't figure out why I added that selector. Removing it now